CHANGELOG.md

```diff
@@ -1,16 +1,22 @@
-# Unreleased Changes
+# Changelog
 
+## Unreleased Changes
 
-# Released Changes
+## Released Changes
 
-## 2.0.0
+### 2.1.0
+
+* Schedule expression variable changed to a rate based expression
```

Comment:
Clarify the it's just the default value that has changed

Reply:
Fixed